qiskit/transpiler/passes/synthesis/high_level_synthesis.py

```diff
@@ -661,6 +666,90 @@ def run(self, high_level_object, coupling_map=None, target=None, qubits=None, **
         return decomposition
 
 
+class QftSynthesisFull(HighLevelSynthesisPlugin):
+    """Synthesis plugin for QFT gates using all-to-all connectivity.
+
+    This plugin name is :``qft.full`` which can be used as the key on
+    an :class:`~.HLSConfig` object to use this method with :class:`~.HighLevelSynthesis`.
+
+    The plugin supports the following additional options:
+
+    * do_swaps (bool): Whether to include Swap gates at the end of the synthesized
```

Comment:
I think that maybe "do swaps" can be confusing, and should be replaced by something like "reverse_qubits"?
also "cna" --> "can"

Reply:
Good catch! I can indeed rename "do_swaps" in these newly added functions. Done!